clarity/src/vm/representations.rs

```diff
@@ -427,15 +427,25 @@ pub struct SymbolicExpression {
 
     #[cfg(feature = "developer-mode")]
     pub span: Span,
+
+    #[cfg(feature = "developer-mode")]
+    pub pre_comments: Vec<(String, Span)>,
+    #[cfg(feature = "developer-mode")]
+    pub end_line_comment: Option<String>,
+    #[cfg(feature = "developer-mode")]
+    pub post_comments: Vec<(String, Span)>,
 }
 
 impl SymbolicExpression {
     #[cfg(feature = "developer-mode")]
     fn cons() -> SymbolicExpression {
         SymbolicExpression {
             id: 0,
-            span: Span::zero(),
             expr: SymbolicExpressionType::AtomValue(Value::Bool(false)),
+            span: Span::zero(),
+            pre_comments: vec![],
+            end_line_comment: None,
+            post_comments: vec![],
```

Comment:
So, I'm _pretty_ sure that altering the `SymbolicExpression` serialization format won't impact a block's consensus hash (because the AST is stored as metadata), but it would be good to have some kind of test fixture-based of this.

There's a couple ways we could go about structuring a test like this, but I think the easiest way to do this would be to add a coordinator-style test. See, for example, the test I add in my atlas PR:

 https://github.com/stacks-network/stacks-blockchain/blob/9b97a39191133a53cfdcb07cccceebbab99246f7/src/chainstate/coordinator/tests.rs#L4312-L4322)

You could run the test first without your changes and get a value for the index block hash. Then, you could use that value as a hardcoded string and add it as an assertion to the test (that way, the test will assert that the commitment hash doesn't change with these changes).